qualification/antenna_channelised_voltage/test_delay.py

```diff
@@ -635,13 +649,7 @@ async def test_group_delay(
     await client.request("delays", "antenna-channelised-voltage", receiver.sync_time, *delay_models)
     pdf_report.detail(f"Set delays to {delay_model} for all inputs.")
 
-    pdf_report.step("Set beamformer weights to use only one antenna.")
-    weights = np.zeros(len(receiver.source_indices[0]))
-    weights[0] = 1.0
-    await client.request("beam-weights", receiver.stream_names[0], *weights)
-    pdf_report.detail(f"Set weights on {receiver.stream_names[0]}.")
-
-    async def measure_once(freqs: tuple[float, float]) -> tuple[float, float, float]:
+    async def measure_once(rel_freqs: tuple[float, float]) -> tuple[float, float, float]:
```

Comment:
So this function's signature is now somewhat confusing and I had to go back and forth a few times to understand what it was doing. 

The way it was being used in the test had me thinking you were doing a test at DC and 12Hz which would obviously not give you meaningful data.

So I'd suggest changing it something like this:
```suggestion
    async def measure_once(frequency_delta: float) -> tuple[float, float, float]:
```

The docstring could probably also use a bit of an update.

Reply:
I'm not following. You've replaces two degrees of freedom (a 2-tuple) with one - how do I reconstruct the original values?

I agree that the docstring could be made more explicit.